src/components/bulk-email-tool/bulk-email-task-manager/BulkEmailContentHistory.jsx

```diff
@@ -1,41 +1,241 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import { useParams } from 'react-router-dom';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 
-import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import {
+  Alert, Button, DataTable, Modal,
+} from '@edx/paragon';
+import messages from './messages';
 import { getSentEmailHistory } from './api';
 
-export default function BulkEmailContentHistory() {
+export function BulkEmailContentHistory({ intl }) {
   const { courseId } = useParams();
 
-  const [emailHistoryData, setEmailHistoryData] = useState(); // eslint-disable-line no-unused-vars
+  const [emailHistoryData, setEmailHistoryData] = useState();
```

Comment:
@Tj-Tracy Justin said you could explain this state handling pattern to me 🙏

Reply:
A link to some documentation that may be helpful as well: https://reactjs.org/docs/hooks-state.html

Reply:
functional components with hooks are how we tend to set up components now, as opposed to class components. `useState` returns a piece of state and the function needed to set the state for it.